kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2609,6 +2619,64 @@ def ceph_host_heartbeat_check(self, req):
+                rsp.qemuConfirmedDead = True
+                logger.info("vm[%s] confirmed dead on suspect host[%s]" % (vm_uuid, target_ip))
+            elif rc == 2:
+                rsp.qemuStillAlive = True
+                rsp.success = False
+                rsp.error = "qemu still alive on %s after force-destroy attempt" % target_ip
+                logger.warn("vm[%s] still alive on suspect host[%s]" % (vm_uuid, target_ip))
+            else:
+                rsp.targetHostUnreachable = True
+                logger.info("ssh to suspect host[%s] failed (rc=%s, err=%s), treat as unreachable"
+                            % (target_ip, rc, err))
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)